test/cli/key.js

```diff
@@ -19,6 +19,7 @@ describe('key', () => runOnAndOff.off((thing) => {
     this.timeout(40 * 1000)
 
     return ipfs(`${pass} key gen ${name} --type rsa --size 2048`)
+      .then(() => ipfs(`${pass} key list -l`))
```

Comment:
This change changes the assertion as now it's testing on the output of `key list` rather that the name is included in the output when running `key gen`

Reply:
`js-ipfs` used to do: 

```console
$ jsipfs key gen foobar
generated Qmb1Wj3YR4PGgscvXJ3JgaiX6GCQFXLwP6Qo7c5kNR6PQK foobar
```

but to bring it inline with `go-ipfs` it now outputs:

```console
$ jsipfs key gen foobar
Qmb1Wj3YR4PGgscvXJ3JgaiX6GCQFXLwP6Qo7c5kNR6PQK
```

so the name is no longer in the output and the test would fail.  In order to verify that we have generated a key with the passed name, I changed the test to examine the output of `jsipfs key list` after generating the key which includes the name.